### DISCUSSION TITLE
Weekly Governance Updates (9 December, 2022)

### DISCUSSION THREAD

**Discussion by luckyparadise** [filecoin-project/FIPs#564] [General]
This weekly report provides a high-level overview of what's happening with Filecoin governance.

Anyone is welcome to participate in governance. To learn more about any of the below materials, simply follow the links and feel free to weigh in.

If you have questions or ideas related to FIPs or Filecoin governance, please start a new discussion in the FIPs Forum or post directly in [#fil-fips](https://github.com/filecoin-project/FIPs/discussions/filecoinproject.slack.com).

---

**Accepted FIPs** - *FIPs awaiting network implementation*

- FIP0027 - Change type of DealProposal Label field from a (Golang) String to a Union [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0027.md)[(Discussion)](https://github.com/filecoin-project/FIPs/issues/187)
- FIP0029 - Beneficiary address for storage providers [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0029.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/253)
- FIP0034 - Set pre-commit deposit independent of sector content [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0034.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/290)
- FIP0041 - Forward Compatibility for PreCommit and ReplicaUpdate [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0041.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/380)
- FIP0044 - Standard Authentication Method for Actors [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0044.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/413)
- FIP0045 - De-couple verified registry from markets [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0045.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/313)
- FIP0047 - Proof Expiration & PoRep Security Policy [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0047.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/415)

**Last Call** - *Final opportunity for community feedback!* 

NA

**Current FIP Drafts** - *Now accepting community feedback; merged into repo*

- FIP0037 - Gas Model Adjustment for User Programmability [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0037.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/316)
- FIP0039 - Filecoin Message Replay Protection [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0039.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/301)
- FIP0048 - f4 Address Class [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0048.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/473)
- FIP0049 - Actor Events [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0049.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/484) 
- FIP0050 - API Between User-Programmed Actors and Built-In Actors [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0050.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/401)
- FIP0052 - Increase max sector commitment to 3.5 years [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0052.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/475)

**Open Pull Requests** - *upcoming FIP drafts awaiting technical review; now accepting community feedback*

- FIP0043 - Introduce Domain Separation - [(FIP)](https://github.com/filecoin-project/FIPs/discussions/387)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/387%20status:%20Draft)
- FIP0051: Adding (Synchronous) Consistent Broadcast to EC - [(related discussion)](https://github.com/filecoin-project/FIPs/discussions/501) 
- First draft of NFT standard- [(PR)](https://github.com/filecoin-project/FIPs/pull/519)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/492)
- Add information about gas increase for FIP0045 - [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0045.md)[(PR)](https://github.com/filecoin-project/FIPs/pull/494)
- Alias informational to FRC - [(PR)](https://github.com/filecoin-project/FIPs/pull/552)
- FRC: non-normative FEVM architectural overview - [(PR)](https://github.com/filecoin-project/FIPs/pull/520)

 **🔥 HOT 🔥 Discussion Topics**

- @anorth put together a post on the gas impacts of FIP-0045. You can read the post in detail [here](https://github.com/filecoin-project/FIPs/discussions/535) 
- @vkalghatgi started a discussion thread aimed at revisiting a Sector Duration Multiplier. Read more about the scenarios being proposed for the SDM Policy. You can read in detail and drop your comments/questions [here](https://github.com/filecoin-project/FIPs/discussions/554)

**Comment by steven004** [comment]
@luckyparadise , 
Some of FIPs in the `Accepted FIPs` part had been implemented in NV17. They need to change to `final` state.
